components/ai_chat/resources/page/chat_ui.tsx

```diff
@@ -122,6 +122,24 @@ function ConversationEntries(props: ConversationEntriesProps) {
     }
   }, [props.onHeightChanged, props.onIsContentReady])
 
+  React.useEffect(() => {
+    // Set the iframe position to relative when the regenerate
+    // answer menu is open. Otherwise the menu can sometimes be
+    // overlapped by the Suggested question buttons.
+    const listener = (isOpen: boolean) => {
+      document.body.style.setProperty(
+        '--iframe-position-for-menus',
+        isOpen ? 'relative' : 'unset',
```

Comment:
hey, could we not just set this to `relative` always and get rid of the mojom interface?